eng/pipelines/mirror-repos.yml

```diff
@@ -91,6 +91,18 @@ jobs:
             TargetRepos:
               Azure/azure-dev-pr:
 
+          Azure/azure-rest-api-specs:
```

Comment:
Duplicate entries will not work as these are key values. You should be able to add to the list in the existing entry above.